jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>Hmmm, I don't see where lowly expressed genes are predicted with less accuracy?? Are you talking about how there's a wider spread for the genes with lower values...? I don't know if you can attribute this to the model, hm. I can imagine that sparsely expressed genes come from a distribution that the model correctly learns, but since both real and simulated data sample sparsely from that distribution, this plot ends up looking noisy like this. But I could be super off here!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='100'/>